trieste/models/gpflow/sampler.py

```diff
@@ -749,18 +763,7 @@ def __init__(
                 f"but received {model.__repr__()}."
             )
 
-        self._kernel = model.get_kernel()
-        self._n_components = n_components
-        super().__init__(self._kernel, self._n_components, dtype=tf.float64)
-
-        if isinstance(model, SupportsGetInducingVariables):
-            dummy_X = model.get_inducing_variables()[0][0:1, :]
-        else:
-            dummy_X = model.get_internal_data().query_points[0:1, :]
-
-        self.__call__(dummy_X)  # dummy call to force init of weights
-        self.b: TensorType = tf.Variable(self.b)
-        self.W: TensorType = tf.Variable(self.W)  # allow updateable weights
```

Comment:
this is now done in GPflux so no need for it?

Reply:
The converting of weights to `tf.Variable` hasn't been needed since [PR 48](https://github.com/secondmind-labs/GPflux/pull/48) in GPflux from Aug 2021. That PR created the weights using Keras layer `add_weight` calls, which creates variables.

The deletion of dummy call explained with the change to `__call__`.